src/github.ts

```diff
@@ -57,7 +63,11 @@ export async function getRequiredChecks(
   repoName: string,
   branchName: string,
 ): Promise<Set<string>> {
-  const branch = await client.repos.getBranch({ owner: organization, repo: repoName, branch: branchName });
+  const branch = await client.repos.getBranch({
```

Comment:
Is your auto-formatter set to a low line-length? I have mine set to 120 to match the linter. It'd be good if you could revert the formatting changes in this file.